src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/DefaultFileUriProvider.cs

```diff
@@ -5,7 +5,7 @@
 using System.Composition;
 using Microsoft.VisualStudio.Text;
 
-namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage
 {
     [Shared]
     [Export(typeof(FileUriProvider))]
```

Comment:
This class has a `TextBufferUri` which stores the associated Uri as `__RazorTextBufferUri`. Should probably change that to be `__TextBufferUri`

Reply:
I started using MsLsp just in case of uniqueness. Let me know if you think we should drop the prefix, I don't have a strong preference, just mild paranoia. :)

---
In reply to: [437126143](https://github.com/dotnet/aspnetcore-tooling/pull/1994#discussion_r437126143) [](ancestors = 437126143)